frontend/packages/dev-console/src/components/helm/HelmReleaseOverview.tsx

```diff
@@ -1,19 +1,28 @@
 import * as React from 'react';
 import { ResourceSummary, SectionHeading } from '@console/internal/components/utils';
 import { K8sResourceKind } from '@console/internal/module/k8s';
+import { HelmRelease } from './helm-types';
+import HelmChartSummary from './HelmChartSummary';
 
 export interface HelmReleaseOverviewProps {
   obj: K8sResourceKind;
+  customData: HelmRelease;
 }
 
-const HelmReleaseOverview: React.FC<HelmReleaseOverviewProps> = ({ obj: resourceDetails }) => {
+const HelmReleaseOverview: React.FC<HelmReleaseOverviewProps> = ({
+  obj: resourceDetails,
+  customData,
+}) => {
   return (
     <div className="co-m-pane__body">
       <SectionHeading text="Helm Release Details" />
       <div className="row">
         <div className="col-sm-6">
           <ResourceSummary resource={resourceDetails} customPathName={'metadata.labels.name'} />
         </div>
+        <div className="col-sm-6">
```

Comment:
nit: will it make sense to have `col-xs-6` it will handle all above cases

Reply:
We are using this in all the details pages. That's why used it to be consistent with all the pages.